pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -218,6 +222,28 @@ func (ef *EventFeed) Start(
 	return nil
 }
 
+// removeDuplicateLogs removes duplicate logs from the list of logs
+// This is needed because some node RPC endpoints can return duplicate logs
+// for a given block range. This is a known bug in FVM and impacts Filecoin
+// and Filecoin Hyperspace nodes.
+// See: https://github.com/filecoin-project/ref-fvm/issues/1350
+func (ef *EventFeed) removeDuplicateLogs(logs []types.Log) []types.Log {
```

Comment:
a new methods is added here that takes care of removing duplicate events before persisting them